src/main/java/Example.java

```diff
@@ -0,0 +1,19 @@
+import org.springframework.boot.SpringApplication;
```

Comment:
Could you move this file to package `ru.mystamps.readnext`? Also it's better to give it a more meaningful name.